<?xml version="1.0" ?>
<testsuite name="ToDo-20230604145023" tests="1" time="18.391" failures="0" errors="1">
	<testcase classname="ToDo" name="test_counter" time="18.391">
		<error type="AssertionError" message="&lt;selenium.webdriver.remote.webelement.Web[102 chars]_4&quot;)&gt; != 1"><![CDATA[Traceback (most recent call last):
  File "/home/magdalena/webapp-test/test_script.py", line 120, in test_counter
    self.assertEqual(counter, 1)
AssertionError: <selenium.webdriver.remote.webelement.Web[102 chars]_4")> != 1
]]></error>
	</testcase>
	<system-out><![CDATA[]]></system-out>
	<system-err><![CDATA[]]></system-err>
</testsuite>
